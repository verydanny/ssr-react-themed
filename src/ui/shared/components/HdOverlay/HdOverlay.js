import React from 'react'
import themed from 'react-themed'
import PropTypes from 'prop-types'

const HdOverlay = ({ theme }) => (
  <div
    data-tid="3dTour"
    className={theme.HdOverlay}
  >
    <span>3D</span> Tour
  </div>
)

HdOverlay.propTypes = {
  theme: PropTypes.object,
}

HdOverlay.defaultProps = {
  theme: {},
}

export default themed(/^HdOverlay/)(HdOverlay)
