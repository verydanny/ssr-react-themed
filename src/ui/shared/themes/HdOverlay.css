.HdOverlay {
  position: absolute;
  background: rgba(0, 0, 0, 0.4);
  color: #fff;
  font-size: 13px;
  line-height: 23px;
  width: 57px;
  height: 23px;
  text-align: center;
  white-space: nowrap;
  right: 238px;
  bottom: 0;
}

.HdOverlay span {
  font-weight: bold;
}
