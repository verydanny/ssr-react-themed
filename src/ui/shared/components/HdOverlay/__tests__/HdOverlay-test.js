import React from 'react'
import { shallow } from 'enzyme'
import HdOverlay from '../HdOverlay'

describe('HdOverlay', () => {
  it('renders', () => {
    const wrapper = shallow(<HdOverlay />)
    expect(wrapper.html()).toMatchSnapshot()
  })
})
