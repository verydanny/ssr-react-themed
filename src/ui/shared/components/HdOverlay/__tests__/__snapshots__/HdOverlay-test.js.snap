// Jest Snapshot v1, https://goo.gl/fbAQLP

exports['HdOverlay renders 1'] = `

<div data-tid="3dTour">
  <span>
    3D
  </span>
  Tour
</div>

`
